composer.json

```diff
@@ -36,7 +36,7 @@
     "require": {
         "php": "^7.1",
         "roave/security-advisories": "dev-master",
-        "zendframework/zend-component-installer": "^1.0",
+        "zendframework/zend-component-installer": "^2.0",
         "zendframework/zend-config-aggregator": "^1.0",
         "zendframework/zend-expressive": "^3.0.0-dev",
         "zendframework/zend-expressive-helpers": "^5.0.0-dev",
```

Comment:
Update this to `^5.0.0alpha2 || ^5.0`, and then:

- each of the template provider entries, including zend-expressive-template, should be `^2.0.0alpha1 || ^2.0`
- each of the router provider entries, including zend-expressive-router, should be `^3.0.0alpha` || ^3.0`

These changes are necessary as otherwise grabbing the lowest supported will give a mix of http-interop and PSR-15 dependencies.